worksheets/how-to-set-up-screen-readers.md

```diff
@@ -24,6 +23,7 @@ The goal of this document is to help Mac users get a virtual machine set up with
 1. Restart your virtual machine
 
 **Pro JAWS tip** [JAWS keyboard shortcuts](https://webaim.org/resources/shortcuts/jaws)
+**Audio Issues?** [VM error "No Audio Output Device is installed"](https://www.robolinux.org/lm/faq/14.html)
```

Comment:
Can we move the audio issues section up under the virtual box section?

Reply:
Sure, if you think that's a better spot. I was torn, but decided on this point since I didn't encounter the issue until I started trying to use JAWS (because I couldn't hear it).

Reply:
I think you would hit the same issue with NVDA if the audio wasn't set up. That's why I was thinking it applies more to the virtual machine set up.

Reply:
100% true. Good point!